source/deploy/postgres-migration.rst

```diff
@@ -6,32 +6,167 @@ Migration guidelines from MySQL to PostgreSQL
+This command outputs the readiness status and prints required fixes for common issues. The flags for fixes are as follows (where all fixes can be used together at the same time):
+
+.. code-block:: shell
+
+   --fix-artifacts               Removes the artifacts from older versions of Mattermost
+   --fix-unicode                 Removes the unsupported unicode characters from MySQL tables
+   --fix-varchar                 Removes the rows with varchar overflow
+
+Step 2 - Create the PostgreSQL database schema
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Run the following command to create the Postgres database schema:
+
+.. code-block:: shell
+
+   migration-assist postgres "<POSTGRES_DSN>" --run-migrations --mattermost-version="<MATTERMOST_VERSION>"
+
+This command downloads the necessary migrations and applies them to the Postgres database. The ``--mattermost-version`` flag is required to specify the Mattermost version you are migrating from.
+
+Step 3 - Generate a pgloader configuration
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Run the following command to generate a pgloader configuration:
+
+.. code-block:: shell
+
+   migration-assist pgloader --mysql="<MYSQL_DSN>" --postgres="<POSTGRES_DSN>" > migration.load
+
+This command will generate a pgloader configuration file that can be used to migrate the data from MySQL to Postgres.
+
+Step 4 - Run pgloader
+^^^^^^^^^^^^^^^^^^^^^^
+
+See the `install pgloader <#install-pgloader>`__ section for installation details.
+
+Run pgloader with the generated configuration file:
+
+.. code-block:: shell
+
+   pgloader migration.load > migration.log
+
+Carefully read the log file to analyze whether there were any errors during the migration process. If there were any errors, please contact Mattermost for further guidance.
+
+Step 5 - Restore full-text indexes
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Run the following command to create the full-text indexes for the ``Posts`` and ``FileInfo`` tables:
+
+.. code-block:: shell
+
+   migration-assist postgres post-migrate "<POSTGRES_DSN>"
+
+This command creates the full-text indexes for the ``Posts`` and ``FileInfo`` tables. Please refer to the `Restore full-text indexes <#restore-full-text-indexes>`__ section for more information.
+
+Step 6 - Complete plugin migrations
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Generate migration configuration for Boards and Playbooks:
+
+.. code-block:: shell
+
+   migration-assist pgloader boards --mysql="<MYSQL_DSN>" --postgres="<POSTGRES_DSN>" > boards.load
+   migration-assist pgloader playbooks --mysql="<MYSQL_DSN>" --postgres="<POSTGRES_DSN>" > playbooks.load
+
+Then run pgloader with the generated configuration files:
+
+.. code-block:: shell
+
+   pgloader boards.load > boards_migration.log
+   pgloader playbooks.load > playbooks_migration.log
+
+Please refer to the `Plugin migrations <#plugin-migrations>`__ section for information on migrating Playbooks and Focalboard.
+
+Manual migration tool recommendations
```

Comment:
Hate to ask @cwarnermm but could this entire section be a child page or can we have both the `migration-assist` page and the "Manual migration" page be child pages. We can create a primary page that explains both paths, links to resources etc.

Reply:
I was thinking similarly, @nab-77, as the current page is long and detailed. I agree with having a landing page that branches off to each path.

Reply:
@nab-77 - My latest commit includes the page split and navigation updates, as well as page redirects for moved content.